cron/worker/main.go

```diff
@@ -168,7 +168,8 @@ func main() {
 			break
 		}
 		if err := processRequest(ctx, req, checksToRun, bucketURL, httpClient, githubClient, graphClient); err != nil {
-			panic(err)
+			// Nack the message so that another worker can retry.
+			subscriber.Nack()
```

Comment:
if you haven't done so already, would be good to set up a dead letter topic (https://cloud.google.com/pubsub/docs/handling-failures#forwarding_to) which lets you set a cap on the number of delivery attempts (to prevent queue blowup if something keeps failing).

Reply:
Done.